components/brave_ads/core/internal/account/wallet/wallet_info.h

```diff
@@ -13,8 +13,9 @@ namespace brave_ads {
 struct WalletInfo final {
   bool IsValid() const;
 
-  bool WasUpdated(const WalletInfo& other) const;
-  bool HasChanged(const WalletInfo& other) const;
+  bool WasUpdated(const WalletInfo& previous) const;
+  bool HasChanged(const WalletInfo& previous) const;
+  bool WasUpdatedFromScratch(const WalletInfo& previous) const;
```

Comment:
nit: `wallet` instead of `previous`

Reply:
The previous argument was `const WalletInfo& other`. Maybe it is better to use it?

Reply:
Changed to `const WalletInfo& other`